components/tray/metadata/tray.yml

```diff
@@ -1,13 +1,19 @@
 name: Tray
-description: Tray Dialogs are typically used to portray information on mobile device or smaller screens.
+description: |
```

Comment:
Should I include information about portrait, landscape orientation and the safe zone here?

https://spectrum.adobe.com/page/tray/

Reply:
I could see examples of this being helpful. Also for VRT.  I think Web Components could also benefit from displaying these cases https://opensource.adobe.com/spectrum-web-components/storybook/?path=/story/tray--default

Reply:
I thought about a way to add examples for portrait and landscape, but there didn't seem to be a simple way since these depend on the [orientation media query](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation) which is determined by the relationship between the viewport height and width.

I meant to include this information in the description.